internal/runbits/requirements/requirements.go

```diff
@@ -246,7 +253,7 @@ func (r *RequirementOperation) ExecuteRequirementOperation(requirementName, requ
 	}
 
 	// refresh or install runtime
-	err = runbits.RefreshRuntime(r.Auth, r.Output, r.Analytics, pj, commitID, orderChanged, trigger, r.SvcModel)
+	err = runbits.RefreshRuntime(r.Auth, r.Output, r.Analytics, pj, strfmt.UUID(commitID), orderChanged, trigger, r.SvcModel)
```

Comment:
Does it make sense to have `bp.StateCommit()` return a `strfmt.UUID`?
Edit: apparently it is? Is this cast still necessary?

Reply:
Whoops, looks like a missed a few of these when I changed the function's return value. Thanks for catching!